tests/crates/enarx_syscall_tests/src/bin/sgx_get_att_quote.rs

```diff
@@ -45,13 +45,14 @@ fn main() -> Result<()> {
     }
 
     if size < expected.len() {
-        return Err(1);
+        return Err(2);
     }
 
-    /* check beginning of quote matches expected value */
-    if !buf.starts_with(&expected) {
-        return Err(1);
-    }
+    /* check beginning of quote matches expected value, after the ASN.1 header */
+    //if !buf[0..8].starts_with(&expected) {
+    //    // New expected value seems to be 030002000000000009000d00939a7233f79c4ca9940a0db3957f0607
```

Comment:
@haraldh Why the change in expected value? Is this a hardware thing? It's only slightly off.

Reply:
How off is it? Can it be that a part of the encoding blends into the beginning of the report? Which bytes are different?